bin/centroid_map

```diff
@@ -126,14 +133,14 @@ def centroid_map(mapfile, img, outfile):
         if 0 == uu:
             continue
         idx = np.where(mapvals == uu)
-        w = np.sum(img.imgvals[idx])
+        w = np.nansum(img.imgvals[idx])
         if 0 == w:
             # If sum is 0, use unweighted value
             cx = np.average(img.xx[idx])
```

Comment:
do we not need to worry about nan's in this case? eg use `np.nanmean` rather than `np.average`?

Reply:
Aha. I hadn't noticed the `img.imgvals` vs `img.xx` difference.